src/store/labelBase.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios"
+
+const GET_DATA = "GET DATA"
+
+const initialCase = {
+  data: [],
+}
+
+const getData = data => ({ type: GET_DATA, data })
+
+export const fetchDataThunk = () => async dispatch => {
+  try {
+    const { data } = await axios.get("/json/index")
```

Comment:
If we have the data be a static JSON file, we can just do an `import { data } from "label.json"` I think.

Reply:
I'm not understanding because my file is under the path I've set. I'm not sure how it would have access to that file if I don't state the directory.